src/scss/02_atoms/_buttons.scss

```diff
@@ -266,22 +269,25 @@ $downloadButtonConfig: (
   padding-left: get($buttonSizeConfig, $size, paddingLeft);
   padding-right: get($buttonSizeConfig, $size, paddingRight);
   padding-top: get($buttonSizeConfig, $size, paddingTop);
+
+  .btn__text {
+    line-height: get($buttonSizeConfig, $size, lineHeight);
+  }
 }
 
 .btn {
   @include buttonSize(base);
-  align-content: center;
-  align-items: center;
-  align-self: center;
   border-style: solid;
   cursor: pointer;
-  display: flex;
+  display: inline-block;
   font-family: font(heavy);
   justify-content: center;
   letter-spacing: rem(0.8px);
+  line-height: 100%;
   outline: none;
   overflow: hidden;
   position: relative;
+  text-align: center;
```

Comment:
Questo causa un problema in `btn--download` (puoi vederlo nel file `buttons.html`)